ct.yaml

```diff
@@ -1,7 +1,5 @@
 # See https://github.com/helm/chart-testing#configuration
-remote: origin
 chart-dirs:
   - charts
-chart-repos:
```

Comment:
The `chart-repos` is an additional chart repository for dependency resolutions and because current Velero helm chart does not have another dependency chart, so we remove it.